src/pyglottolog/links/endangeredlanguages.py

```diff
@@ -10,10 +10,62 @@
+    'hach1239': 10378, 'hams1239': 10109, 'hare1244': 2442, 'hill1258': 10414, 'huzh1238': 2088,
+    'hwar1238': 6038, 'hwel1241': 6106, 'ikor1238': 8194, 'indo1291': 7185, 'jaba1234': 10839,
+    'jarq1234': 10074, 'jows1234': 10722, 'jude1271': 10502, 'jude1276': 10499, 'kais1242': 3214,
+    'kamd1238': 5998, 'kani1276': 6688, 'kark1255': 5641, 'kart1247': 6689, 'kash1276': 10500,
+    'kaur1267': 5871, 'kesh1234': 10724, 'kham1281': 8234, 'khap1242': 5884, 'khuf1238': 6108,
+    'komi1277': 3280, 'komn1238': 10748, 'kuhp1234': 10073, 'kuma1284': 10475, 'lang1309': 3391,
+    'lang1321': 552, 'lang1332': 10463, 'lara1261': 8116, 'lawi1235': 6045, 'lenc1243': 7256,
+    'leva1238': 5657, 'limo1246': 10461, 'liya1241': 5562, 'liya1242': 5549, 'lizu1234': 10624,
+    'long1406': 10814, 'lopn1238': 10481, 'lush1256': 10462, 'maan1239': 5194, 'mand1418': 5563,
+    'mans1258': 8529, 'many1256': 6696, 'mard1245': 2053, 'mayy1239': 5883, 'mbiy1238': 6700,
+    'meym1234': 10071, 'midd1324': 1751, 'mila1245': 1378, 'minh1238': 10471, 'miny1239': 7172,
+    'mith1236': 6704, 'mpra1235': 3725, 'mudb1240': 2550, 'muya1239': 5609, 'mwen1238': 1135,
+    'mwin1241': 5049, 'nasv1234': 10789, 'nati1244': 2512, 'navw1234': 3065, 'nene1249': 2062,
+    'nese1235': 2089, 'ngal1294': 5546, 'ngum1253': 6717, 'nisv1234': 10791, 'niti1249': 10795,
+    'norm1245': 3284, 'nort3264': 8550, 'nucl1693': 10453, 'nyan1311': 534, 'oldb1247': 7187,
+    'ongk1234': 8571, 'ordo1245': 10470, 'orig1234': 7006, 'orko1234': 10801, 'oros1238': 5866,
+    'pada1257': 5677, 'pail1243': 7171, 'pena1270': 8741, 'phal1255': 5938, 'phon1243': 5874,
+    'phut1246': 10746, 'picu1248': 6509, 'plai1258': 9011, 'poit1240': 3269, 'prov1235': 8629,
+    'pudi1238': 5451, 'ramo1243': 5684, 'rats1237': 10309, 'rucc1239': 5876, 'ruku1239': 486,
+    'rush1239': 5870, 'sadu1234': 10708, 'sakh1247': 2896, 'sama1296': 5173, 'sapw1237': 10424,
+    'sede1245': 10847, 'shee1239': 5932, 'shim1250': 5691, 'shir1258': 10503, 'shir1268': 5048,
+    'sich1239': 5811, 'sigi1234': 10495, 'solo1263': 8570, 'sout2962': 2091, 'sout3226': 8551,
+    'sout3262': 8555, 'soyo1234': 10480, 'sure1238': 5695, 'taiw1247': 2805, 'tana1291': 8118,
+    'tang1377': 7170, 'tazh1234': 5180, 'tazz1244': 8553, 'teja1235': 3114, 'tets1235': 8509,
+    'teus1236': 2150, 'toum1239': 5872, 'tsix1234': 4570, 'tuha1234': 10479, 'tyem1238': 5947,
+    'urim1252': 10568, 'urmi1249': 9289, 'viva1235': 945, 'vivt1234': 10794, 'vovo1238': 10797,
+    'wail1244': 2174, 'wang1288': 6269, 'wara1302': 2707, 'wurl1240': 5452, 'xinc1242': 9709,
+    'xinc1246': 6195, 'xink1235': 6196, 'yang1305': 5560, 'yara1254': 6806, 'yaww1238': 7173,
+    'yazd1240': 10501, 'yila1234': 10717, 'yogy1234': 7193, 'yulp1239': 5245, 'yuwa1242': 5987,
+    'zaks1239': 10489, 'zara1252': 10444, 'zefr1234': 10072, 'zemi1238': 1253, 'zoro1244': 1630,
+    'zull1239': 10464,
+}
```

Comment:
Didn't we talk about ~135 missing matches? This looks more like 250.

Reply:
yes - the list was extended due to the fact that ElCat's iso field lists many empty cells and non-valid ISO codes, thus I added them as well. There are still 62 ElCat IDs which I couldn't map against Glottolog. This list I can provide easily.